packages/just-scripts/src/tasks/sassTask.ts

```diff
@@ -61,7 +61,7 @@ export function sassTask(
                   const css = result.css.toString();
 
                   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                  postcss([autoprefixerFn, ...postcssPlugins!])
+                  postcss([autoprefixerFn, postcssRtlFn, ...postcssPlugins!, cssnano()])
```

Comment:
i'm little worried here , what if the new plugins we added as default here are defined in `postcssPlugins` already. can we check ?

Reply:
@dzearing & I chatted offline on this point - we're going to have a "final postcss plugins" array that will optionally include these if those plugins are detected (progressive enhancement of postcss plugins!)

Reply:
We can't check because these functions are created new for each plugin. Plugins are factories for the functions, so we can't do reference compares.

I'd suspect we'd run it twice. I don't see this being very problematic as plugins should be idempotent. It would simply mean slower builds.